plugin/pkg/admission/runtimeclass/admission_test.go

```diff
@@ -155,6 +166,127 @@ func TestSetOverhead(t *testing.T) {
 		})
 	}
 }
+
+func TestSetScheduling(t *testing.T) {
+	tests := []struct {
+		name         string
+		runtimeClass *v1beta1.RuntimeClass
+		pod          *core.Pod
```

Comment:
Add a testcase with a nil nodeSelector on the pod.

Reply:
done